cnd/src/swap_protocols/rfc003/events/ledger_event_futures.rs

```diff
@@ -35,31 +36,43 @@ impl<L: Ledger, A: Asset> LedgerEventFutures<L, A> {
 }
 
 impl<L: Ledger, A: Asset> LedgerEvents<L, A> for LedgerEventFutures<L, A> {
-    fn htlc_deployed(&mut self, htlc_params: HtlcParams<L, A>) -> &mut DeployedFuture<L> {
+    fn htlc_deployed(
+        &mut self,
+        htlc_params: HtlcParams<L, A>,
+        timestamp: NaiveDateTime,
```

Comment:
It would be nice if this variable would say more about _what_ this timestamp represents :)

Reply:
Yep I was hoping to have this discussion here, also I wanted to do the linking of the timestamp with the db first then the rename so review was easier.  I will add an patch doing the rename then everyone interested can chime in on the name.

Reply:
So, FTR I used the following names:
- In the db (when the accept message is inserted/read) we call the timestamp `at`.
- In the state machine (in `OngoingSwap` we call it `accepted_at`)
- Within btsieve where the timestamp is used to go back in time, call it `after` - since we are checking if a block was mined 'after' the time.

Comments/improvements @comit-network/rust-devs

Reply:
Top names! I like them :)